.evergreen/run-tests.sh

```diff
@@ -172,6 +172,7 @@ if [ -n "$TEST_ENCRYPTION" ]; then
 
     # Get access to the AWS temporary credentials:
     # CSFLE_AWS_TEMP_ACCESS_KEY_ID, CSFLE_AWS_TEMP_SECRET_ACCESS_KEY, CSFLE_AWS_TEMP_SESSION_TOKEN
+    export AWS_SESSION_TOKEN=
```

Comment:
Is this a bug fix?

Reply:
Previously we did this as part of the encryption env setup block. I moved it here for clarity since it's required before the next line.